platform/elrond/transaction.go

```diff
@@ -25,6 +25,10 @@ func NormalizeTxs(srcTxs []Transaction, address string, block Block) (txs []type
 
 // NormalizeTx converts an Elrond transaction into the generic model
 func NormalizeTx(srcTx Transaction, address string, block Block) (tx types.Tx, ok bool) {
+	if srcTx.HasNegativeValue() {
```

Comment:
you mentioned that this only happens for smart contract transactions, maybe better would be to avoid processing smart contract transactions and only allow transfers?